app/src/main/java/lv/spkc/apturicovid/activity/BaseActivity.kt

```diff
@@ -261,6 +304,14 @@ abstract class BaseActivity : DaggerAppCompatActivity() {
         }
     }
 
+    private fun checkRequiredServicesState() {
+        lifecycleScope.launch(CovidCoroutineExceptionHandler("Error checking required services state")) {
+            val isOperational = (this@BaseActivity.getSystemService(Context.LOCATION_SERVICE) as? LocationManager)?.let { LocationManagerCompat.isLocationEnabled(it) } ?: false
```

Comment:
The amount of characters in this line is too damn high!

Perhaps extract `val locationManager = (this@BaseActivity.getSystemService(Context.LOCATION_SERVICE) as? LocationManager)`